openrtb_ext/request_wrapper.go

```diff
@@ -46,7 +46,7 @@ const (
 	jsonEmptyObjectLength               = 2
 	consentedProvidersSettingsStringKey = "ConsentedProvidersSettings"
 	consentedProvidersSettingsListKey   = "consented_providers_settings"
-	consentKey                          = "consent"
```

Comment:
We should be keeping this as `consentKey` to be consistent with other keys listed below.

Reply:
Got changed by mistake. Reverted the change